src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj

```diff
@@ -86,8 +86,9 @@
 		<PackageReference Include="ReviewService.NativePrompters" Version="1.0.0" />
 		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
 		<PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
-		<PackageReference Include="Uno.Material.WinUI" Version="4.0.6" />
-		<PackageReference Include="Uno.Toolkit.WinUI.Material" Version="5.0.17" />
+		<PackageReference Include="Uno.Material.WinUI" Version="4.2.0-dev.18" />
```

Comment:
Can we use stable (non pre-release) instead?

Reply:
i closed this one an reopened on the Fork one